app/controllers/features_controller.rb

```diff
@@ -47,6 +48,11 @@ def disable_feature
             end
     feature = params[:feature]
     authorize actor
+
+    if feature == "sudo_mode_2015_07_21"
+      return unless enforce_sudo_mode # rubocop:disable Style/SoleNestedConditional
+    end
```

Comment:
just to sanity check - this is requiring people to pass sudo mode checks to enable sudo mode?

Reply:
If so, i'm not sure if this works as expected since `enforce_sudo_mode` and `sudo_mode?` will always return true because of:

```ruby
def sudo_mode?
    return true unless Flipper.enabled?(:sudo_mode_2015_07_21, user)
    # ...
```